### PR TITLE
Update remove imported dialog props to support principal only

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

In the case of a failed imported token, we don’t have the token metadata, such as the logo and token name, but users should still be able to remove the imported token. To achieve this, we first need to make the confirmation dialog work with the ledger canister ID only.

# Changes

- Change the confirmation component props to work seamlessly with either the universe or the ledger canister ID.

# Tests

Updated.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/tests/lib/components/accounts/ImportTokenRemoveConfirmation.spec.ts [hunk 1]
```diff
@@ -4,60 +4,83 @@ import { principal } from "$tests/mocks/sns-projects.mock";
 import { ImportTokenRemoveConfirmationPo } from "$tests/page-objects/ImportTokenRemoveConfirmation.page-object";
 import { JestPageObjectElement } from "$tests/page-objects/jest.page-object";
 import { render } from "$tests/utils/svelte.test-utils";
+import type { Principal } from "@dfinity/principal";
 
 describe("ImportTokenRemoveConfirmation", () => {
-  const ledgerCanisterId = principal(1);
+  const ledgerCanisterId1 = principal(0);
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/accounts/ImportTokenRemoveConfirmation.spec.ts:10]: No, there was no specific reason, this was checked out from another branch. These changes are not required at all, so they were reverted.

[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/accounts/ImportTokenRemoveConfirmation.spec.ts:10]: Answered in the comment above.

File: frontend/src/lib/components/accounts/IcrcWalletPage.svelte [hunk 2]
```diff
@@ -21,7 +21,7 @@
   } from "$lib/utils/accounts.utils";
   import { replacePlaceholders } from "$lib/utils/i18n.utils";
   import { IconDots, Island, Spinner } from "@dfinity/gix-components";
-  import type { Principal } from "@dfinity/principal";
```
Comment (dskloetd) [inline frontend/src/lib/components/accounts/IcrcWalletPage.svelte:24]: I don't see any change in this file that would require changing the way `Principal` is imported. Why is this necessary? If it's not, I'm surprised it didn't cause a lint error.

File: frontend/src/lib/components/accounts/ImportTokenRemoveConfirmation.svelte [hunk 3]
```diff
@@ -2,11 +2,13 @@
   import { i18n } from "$lib/stores/i18n";
   import UniverseSummary from "$lib/components/universe/UniverseSummary.svelte";
   import type { Universe } from "$lib/types/universe";
-  import { nonNullish } from "@dfinity/utils";
   import ConfirmationModal from "$lib/modals/common/ConfirmationModal.svelte";
   import { Tag } from "@dfinity/gix-components";
+  import { Principal } from "@dfinity/principal";
```
Comment (dskloetd) [inline frontend/src/lib/components/accounts/ImportTokenRemoveConfirmation.svelte:7]: Can `Principal` be imported as just a type here as well?

**Review by dskloetd — APPROVED** [review]

File: frontend/src/lib/components/accounts/IcrcWalletPage.svelte (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/accounts/IcrcWalletPage.svelte]: I’m also surprised by the linter. Reverted.

File: frontend/src/lib/components/accounts/ImportTokenRemoveConfirmation.svelte (hunk 3 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/accounts/ImportTokenRemoveConfirmation.svelte]: Weird. Done.
